turbopack/crates/turbopack-core/src/environment.rs

```diff
@@ -303,7 +304,9 @@ impl NodeJsEnvironment {
 
 #[turbo_tasks::value(shared)]
 pub enum NodeJsVersion {
+    /// Use the version of Node.js that is available from the environment (via `node --version`)
     Current(ResolvedVc<Box<dyn ProcessEnv>>),
+    /// Use the specified version of Node.js.
     Static(ResolvedVc<RcStr>),
```

Comment:
nit:given that this is coming from a global constant, storing it is a vc is probably not needed? just store the rcstr directly?